types/json-ptr/json-ptr-tests.ts

```diff
@@ -0,0 +1,52 @@
+import JsonPointer from 'json-ptr';
```

Comment:
Use the commonjs syntax `import JsonPointer = require('json-ptr')`.

Consumers of the package who want to use `import JsonProinter from 'json-ptr'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.

Reply:
Done. 🙂